runtime/distribution/build.gradle.kts

```diff
@@ -71,6 +71,17 @@ distributions {
             into("bin") {
                 from("bin/server")
                 from("bin/admin")
+                from("${rootDir}/polaris")
+            }
+
+            into("bin/client") {
+                from("${rootDir}/client/python") {
+                    include("**/*")
+                }
+            }
+
+            into("regtests") {
+                from("${rootDir}/regtests/requirements.txt")
             }
```

Comment:
It's weird to have this directory in the binary distribution. We can make a symlink in the dir `client/polaris`, and point to it instead.